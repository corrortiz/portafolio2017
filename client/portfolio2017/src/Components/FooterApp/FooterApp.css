html {
  height: 100%;
  box-sizing: border-box;
}
html .before .after {
  box-sizing: inherit;
}

.footer {
  position: absolute;
  right: 0;
  bottom: 0;
  left: 0;
  padding: 1em;
  background-color: #000000;
  text-align: center;
  color: white;
  font-family: 'Roboto', 'Helvetica', 'Arial', sans-serif;
}
