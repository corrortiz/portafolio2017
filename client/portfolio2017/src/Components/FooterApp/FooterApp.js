import React from 'react';
//Internals Component
import AppSnackBar from '../AppSnackBar/AppSnackBar';
import Github from '../../images/SVG/Github';

/**
 * Componet for the layout of the footer
 */
const FooterApp = () => {
  return (
    <footer className="footer">
      <div className="footer__divider">
        <h4 className="footer__title">AO HyS PortFolio</h4>
        <a
          href="https://github.com/corrortiz"
          target="blank"
          className="footer__hyperlink"
        >
          <Github />
        </a>
        <AppSnackBar />
      </div>
    </footer>
  );
};

export default FooterApp;
