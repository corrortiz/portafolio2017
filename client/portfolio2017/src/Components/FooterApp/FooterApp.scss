footer{
  flex-shrink: 0;
  padding: 3rem 0;
  font-size: 1.4 rem;
  background-color: #000000;
  text-align: center;
  color: white;
}

.android{
  margin-top: 20px;
  display: block;
}