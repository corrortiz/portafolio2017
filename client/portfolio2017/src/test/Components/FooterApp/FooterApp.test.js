// @flow
import React from 'react';
import { shallow } from 'enzyme';
import FooterApp from '../../../Components/FooterApp/FooterApp';

describe('FooterApp Component', () => {
  it('shold compara a snapshot of FooterApp', () => {
    const wrapper = shallow(<FooterApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
