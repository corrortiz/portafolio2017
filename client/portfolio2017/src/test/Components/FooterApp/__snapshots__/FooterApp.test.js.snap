// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterApp Component shold compara a snapshot of FooterApp 1`] = `
<footer
  className="footer"
>
  <div
    className="footer__divider"
  >
    <h4
      className="footer__title"
    >
      AO HyS PortFolio
    </h4>
    <a
      className="footer__hyperlink"
      href="https://github.com/corrortiz"
      target="blank"
    >
      <Github />
    </a>
    <Connect(NameWrappedComponent) />
  </div>
</footer>
`;
